### ISSUE TITLE
Support for S38 Sonos One SL

### ISSUE THREAD

**Issue by 0stone0** [duncan3dc/sonos#122]
The `isSpeaker()` function does not support the Sonos ONE SL speaker.

Are there any plans of adding support for this speaker?

```
<manufacturer>Sonos, Inc.</manufacturer>
<manufacturerURL>http://www.sonos.com</manufacturerURL>
<modelNumber>S38</modelNumber>
<modelDescription>Sonos One SL</modelDescription>
<modelName>Sonos One SL</modelName>
```

**Comment by larascasse** [comment]
hello,
At the same time, could you add support for the **Sonos Roam SL - S35** ?
Thank you!

**Comment by larascasse** [comment]
In my system, the ONE SL is S22 ...

 <modelNumber>S22</modelNumber>
  <modelDescription>Sonos One SL</modelDescription>
    <modelName>Sonos One SL</modelName>

**Comment by dutchie027** [comment]
[Here is an older Sonos Community](https://en.community.sonos.com/advanced-setups-229000/sonos-and-the-fcc-6796981) post showing a lot of the "model" numbers.

**Comment by duncan3dc** [comment]
All these models should be available in [2.2.1](https://github.com/duncan3dc/sonos/releases/tag/2.2.1) now, thanks
